CHANGELOG.md

```diff
@@ -8,8 +8,10 @@
 
 ### Language support:
 
-* **Vagrantfile & Jenkinsfile**: Support for Jenkinsfile & Vagrantfile syntax highlighting.
+* **Vagrantfile, Jenkinsfile, and Rakefile**: Support for Jenkinsfile, Vagrantfile and Rakefile syntax highlighting.
   [pull request #174](https://github.com/refined-bitbucket/refined-bitbucket/pull/174).
+  resolves [issue #184](https://github.com/refined-bitbucket/refined-bitbucket/issues/184).
```

Comment:
Please include the link for the current pull request as well!

`[pull request #186](https://github.com/refined-bitbucket/refined-bitbucket/pull/186)`

Reply:
Yeah I was actually wondering if there was a way to add this before actually opening the PR, is it possible? Of course I can add it now very easily, I'm mostly wondering for any future PRs.

Reply:
Well I usually push the CHANGELOG change in a later commit after I've already opened the PR and that's how I know, but you could also check the most recently opened issue or PR and just add 1 to the number since ids are generated sequentially 😅 .